### PR TITLE
fix(custom templates): Correct the logic to check for custom templates and load them if necessary.

### PR DESCRIPTION
**Pull request by Camwyn**
Custom templates are chosable in the admin but are not used on the FE.

📷 http://p.tri.be/TmvjQn

[TEC-3253]

[TEC-3253]: https://moderntribe.atlassian.net/browse/TEC-3253

### CODE REVIEW

File: src/Tribe/Views/V2/Template_Bootstrap.php [hunk 1]
```diff
@@ -69,39 +69,42 @@ public function disable_v1() {
...
+	 * @since TBD specifically pass 'event' to be clearer down the line where we check for custom templates.
 	 *
-	 * @return string Either 'event' or 'page' based templates
+	 * @return string Either 'event', 'page' or custom based templates
 	 */
 	public function get_template_setting() {
-		$template = 'event';
 		$default_value = 'default';
-		$setting = tribe_get_option( 'tribeEventsTemplate', $default_value );
+		$template = tribe_get_option( 'tribeEventsTemplate', $default_value );
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Template_Bootstrap.php]: :warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space

<sub>hash: ca8b2028e2f7da25ed9d95fb0f47e713</sub>

File: src/Tribe/Views/V2/Template_Bootstrap.php [hunk 2]
```diff
@@ -69,39 +69,42 @@ public function disable_v1() {
...
 
-		if ( $default_value === $setting ) {
+		if ( empty( $template )  ) {
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Template_Bootstrap.php]: :no_entry_sign: Expected exactly one space before closing parenthesis; "  " found.

<sub>hash: 9b5d58c79d22578222c8a8bf1e038aad</sub>

File: src/Tribe/Views/V2/Template_Bootstrap.php [hunk 3]
```diff
@@ -69,39 +69,42 @@ public function disable_v1() {
...
+			$template = 'event';
+		} elseif( $default_value === $template ) {
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Template_Bootstrap.php]: * :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSEIF keyword; 0 found
* :no_entry_sign: There must be a space between elseif and the first parenthesis '('

<sub>hash: 2417010dbb73032419e22b913158a132</sub>

File: src/Tribe/Views/V2/Template_Bootstrap.php [hunk 4]
```diff
@@ -69,39 +69,42 @@ public function disable_v1() {
...
 	 *
 	 * @return object
 	 */
 	public function get_template_object() {
 		$setting = $this->get_template_setting();
 
-		return $setting === 'page'
-			? tribe( Template\Page::class )
-			: tribe( Template\Event::class );
+		return $setting === 'event'
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Template_Bootstrap.php]: :no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 6950f59a2bd5072b0f812b951c02ce76</sub>

**Comment by jesseeproductions** [comment]
Adding myself to help push this along. Fixed the tests to pass. Not sure if the invalid and numeric template options should be tested now.

**Pull request merged**